src/lib/queries/lexical-query.js

```diff
@@ -28,14 +28,16 @@ export default class LexicalQuery extends Query {
     while (true) {
       if (!this.active) { this.finalize() }
       if (Query.isPromise(result.value)) {
-        try {
-          let resolvedValue = await result.value
-          result = iterator.next(resolvedValue)
+        // try {
```

Comment:
is it no longer possible to get an error thrown here?

Reply:
sorry - I have forgotten to uncomment!